### ISSUE TITLE
no ppm info in channel results?

### ISSUE THREAD

**Issue by imranrajjad** [ashmastaflash/kal-wrapper#2]
hi,

this is a great wrapper on top of Kal, however I was unable to find the ppm value in channel results. Is there any way to calculate PPM using those results?
thanks
Imran

**Comment by ashmastaflash** [comment]
Let me see what I can do about that...

**Comment by ashmastaflash** [comment]
Hi @imranrajjad 
Clone down and install what's in the master branch now and tell me if it works for you.  I added "avg_absolute_error" to the output of each list item returned scan_channel().  If it works to your satisfaction, I'll cut a new package for PyPI and close this issue.

**Comment by imranrajjad** [comment]
thanks a lot..avg_absolute_error exists in all iterations but I am guessing I should take from first iteration

**Comment by ashmastaflash** [comment]
That's correct.  I'm considering changing the output from that mode to a dictionary that more closely represents the output format of the kal binary itself.

**Comment by ashmastaflash** [comment]
Hey @imranrajjad, I know it's been a while...
I re-structured the output from the channel scan feature, and I'm about to cut a new version (v2). 
I would love to get your feedback!
